### PR TITLE
tentative review of setup.py

### PR DESCRIPTION
**Pull request by parmentelat**
for using only setuptools, and simplifying as much as possible

at the same time this would take care of #357

Damian please let me know what you think about going in this direction  
I feel like I may be missing something here, so I don't want to spend more time until you give me the green light, in case this is a dead end

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,63 +1,75 @@
...
+    if target_dir == 'share/jupyter/nbextensions/rise':
+        files.append('README.md')
     data_files.append((target_dir, files))
+
 data_files.append(('etc/jupyter/nbconfig/notebook.d', ['rise.json']))
 
-version_ns = {}
-with open(pjoin(here, name, '_version.py')) as f:
-    exec(f.read(), {}, version_ns)
+data_files
```
Comment (damianavila) [inline setup.py]: What is the purpose of this line?

File: setup.py [hunk 2]
```diff
@@ -1,63 +1,75 @@
...
+
+INSTALL_REQUIRES = [
+    'notebook>=5.5',
+]
 
 with open('./README.md') as readme:
     README = readme.read()
 
+for d in data_files:
```
Comment (damianavila) [inline setup.py]: debugging stuff? If you really want to show the content of the data_files object, I would put this piece after line 51, I think...

**Comment by damianavila** [comment]
@parmentelat, I left some minor comments, I think this is going in the right direction. Do you want to push further on this? Otherwise, I will merge it after your answers/clarifications (and after testing the whole installation with this change).

**Comment by parmentelat** [comment]
oops it looks like I screwed up here, because I used the name 'master' in my own repo, which I thought was to be in line with yours

in any case closing this was unintentional, let me fix this in my repo first

**Comment by parmentelat** [comment]
look Damian, I am terribly sorry; I've been able to resurrect the commit in question, but no matter how hard I tried, I have not yet been able to retrieve your comments on this PR

I am going to have to create a new PR with the exact same contents; I'll keep on looking for a way to resurrect your comments but am not very optimistic

deepest apologies again for that screw up;
unfortunately I had not taken the time to push this under a new name, and had been using 'master'
in the meanwhile I totally forgot about that, and my mental model was that 'master' on my own repo was safe to force-push; it was not, obviously :(

**Comment by damianavila** [comment]
> I am going to have to create a new PR with the exact same contents; I'll keep on looking for a way to resurrect your comments but am not very optimistic

No problem.

> deepest apologies again for that screw up;

No need to apologize.

**Pull request closed without merging**